ingestion/src/metadata/ingestion/stage/migrate.py

```diff
@@ -1,127 +0,0 @@
-#  Copyright 2021 Collate
```

Comment:
Only used for migration from 0.9 to 0.10. Deprecating it as the implementation won't even work with current entities